docs/man/rpm-config.5.scd

```diff
@@ -218,7 +218,7 @@ for example:
 ```
 
 ## Hash algorithms
-The following hash algorithms (denoted by _HASHALGO_) are known to rpm,
+The following hash algorithms (denoted by _HASHALGOS_) are known to rpm,
```

Comment:
Not fully sold on this, we could have a single `_HASHALGO_` reference somewhere else in the future...

Reply:
Yup this doesn't seem right, `_HASHALGO_`  represents a single hash algorithm. pkgverify_digests takes a list of them, but that's the exception. The other items are on build-side though.

Reply:
Ack, reverted that now.